geodepy/fileio.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+"""
+Geoscience Australia - Python Geodesy Package
+Constants Module
```

Comment:
Small comment - update comment text to 'File Input/Output Module'

Reply:
Fixed. Pushing changes now